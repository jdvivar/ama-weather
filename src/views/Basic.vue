<template>
  <div class="basic">
    <div class="wrapper">
      <TypeSelect />
      <div class="line">
        Forecast
      </div>
      <TimeSelect />
      <div class="line">
        In
      </div>
      <LocationSelect />
    </div>
    <BasicWeather />
  </div>
</template>

<script>
import TypeSelect from '@/components/TypeSelect.vue'
import TimeSelect from '@/components/TimeSelect.vue'
import LocationSelect from '@/components/LocationSelect.vue'
import BasicWeather from '@/components/BasicWeather.vue'

export default {
  name: 'basic',
  components: {
    TypeSelect,
    TimeSelect,
    LocationSelect,
    BasicWeather
  }
}
</script>

<style lang="scss">
@import '../styles/line';
@import '../styles/wrapper';
</style>
