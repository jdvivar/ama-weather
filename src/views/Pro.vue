<template>
  <div class="pro">
    <div class="wrapper">
      <TypeSelect />
      <div class="line">
        Weather
      </div>
      <div class="line">
        In
      </div>
      <LocationSelect />
    </div>
    <ProWeather />
  </div>
</template>

<script>
import TypeSelect from '@/components/TypeSelect'
import WhenSelect from '@/components/WhenSelect'
import LocationSelect from '@/components/LocationSelect'
import ProWeather from '@/components/ProWeather'

export default {
  name: 'pro',
  components: {
    TypeSelect,
    WhenSelect,
    LocationSelect,
    ProWeather
  }
}
</script>
