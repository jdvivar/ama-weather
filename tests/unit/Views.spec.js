import { shallowMount } from '@vue/test-utils'
import Basic from '@/views/Basic'
import Pro from '@/views/Pro'

describe('Views', () => {
  it('Basic view has correct markup', () => {
    const wrapper = shallowMount(Basic)
    expect(wrapper.html()).toMatchSnapshot()
  })
  it('Pro view has correct markup', () => {
    const wrapper = shallowMount(Pro)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
