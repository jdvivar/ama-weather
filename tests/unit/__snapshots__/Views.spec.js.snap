// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Views Basic view has correct markup 1`] = `
<div class="basic">
  <div class="wrapper">
    <typeselect-stub></typeselect-stub>
    <div class="line">
      Forecast
    </div>
    <whenselect-stub></whenselect-stub>
    <div class="line">
      In
    </div>
    <locationselect-stub></locationselect-stub>
  </div>
  <basicweather-stub></basicweather-stub>
</div>
`;

exports[`Views Pro view has correct markup 1`] = `
<div class="pro">
  <div class="wrapper">
    <typeselect-stub></typeselect-stub>
    <div class="line">
      Weather
    </div>
    <div class="line">
      In
    </div>
    <locationselect-stub></locationselect-stub>
  </div>
  <proweather-stub></proweather-stub>
</div>
`;
